### PR TITLE
Iteration0/set up comp

### PR DESCRIPTION
**Pull request by saraho1123**
## Is this a fix or a feature?
This is our html and css to match the comp

## What is the change?
Completed the comp

## What does it fix? 
n/a

## Where should the reviewer start?
line 1 of index.html
line 1 of styles.css

We kept the basic colors and fonts commented out at the top of the CSS for easy access in the future.

## How should this be tested?
-open index.html

### Screenshot (optional)

@sarah01123
@elle624
@atownse

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,39 +15,41 @@ <h3>New Activity</h3>
         <div class="selection-box">
           <h4>Select a category:</h3>
           <article class="categories">
-            <button class="study" type="button" name="button">
-              <img class"study-img" src="" alt="">
-              <h4>Study</h4>
+            <button class="study cat-button" type="button" name="button">
```
Comment (yiranli624) [inline index.html:18]: Looking Great!!

**Review by yiranli624 — COMMENTED** [review]
We are off to a great start!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            </div>
+          </article>
+          <div class="start-activity">
+            <button class="start-activity-button" type="button" name="start-activity">START ACTIVITY</button>
+          </div>
+        </div>
+      </section>
+      <section class="past-activities">
+        <h3>Past Activities</h3>
+        <p class="hidden">You haven't logged any activites yet.<br>Complete the form to the left to get started! </p>
```
Comment (atownse) [inline index.html:52]: Watch the hidden here, don't want this stuck invisible

**Review by atownse — COMMENTED** [review]
Maybe update images so they're smaller, might not need to

**Pull request merged**